recipes/suitcase-nomad-camels-hdf5/meta.yaml

```diff
@@ -33,7 +33,7 @@ test:
     - suitcase.nomad_camels_hdf5
     - suitcase.nomad_camels_hdf5.tests
   commands:
-    # - pip check
+    # - pip check  # bluesky reports version numnber 0.0.0 so this check is disabled
```

Comment:
`Looks like https://github.com/conda-forge/bluesky-feedstock/blob/main/recipe/meta.yaml needs 
setuptools-scm >=6.2`. Can you open an issue/PR about that ? If we keep disabling pip-check we create a cascate of skips instead of proper fixes.